apps/wallet/src/ui/app/pages/initialize/create/index.tsx

```diff
@@ -5,12 +5,11 @@ import { Formik, Form, Field } from 'formik';
 import { useNavigate } from 'react-router-dom';
 
 import { createMnemonicValidation } from './validation';
-import Button from '_app/shared/button';
+import { Button } from '_app/shared/ButtonUI';
```

Comment:
I know you didn't name this, but can we shove the old component somewhere like `/shared/deprecated` or `/shared/legacy` so that we don't have button and ButtonUI?

Reply:
Good point but I think only a few places left and probably this PR removes all of the usages of the previous one. So we could just remove the old one and rename the new to just Button

Reply:
@Jordan-Mysten I think I'll save that for a future PR, if that's okay (both deprecating and renaming). I'm pretty sure I replaced all the old ones, and I don't want to bloat this one too much more.